proto/viam/service/slam/v1/slam.proto

```diff
@@ -128,4 +128,17 @@ message GetPropertiesRequest {
 message GetPropertiesResponse {
   bool cloud_slam = 1;
   MappingMode mapping_mode = 2;
+  string internal_state_file_type = 3;
+  repeated SensorInfo sensor_info = 4;
```

Comment:
Double checking if these should be optional?

Reply:
We'll need `sensor_info` to create the replay sensors. I think we could make `internal_state_file_type` optional. 

cc @JohnN193